python/foglamp/services/common/microservice.py

```diff
@@ -67,8 +77,8 @@ def __init__(self, default_config):
             })
             self._core_microservice_management_client.create_configuration_category(config_payload)
```

Comment:
L#68 comment move here and Add below code line for parent/child relationship and comment it, so post merge of 1432 below code works
> self._core_microservice_management_client.create_child_category("General", ["Security"])